DevOps/frankfurter/scripts/install/master_frank_setup.sh

```diff
@@ -51,25 +53,41 @@ if [ ! -f $PIECENTRAL_DIR/runtime/hibikeDevices.csv ]; then
   ln -s $PIECENTRAL_DIR/hibike/hibikeDevices.csv $PIECENTRAL_DIR/runtime/testy/hibikeDevices.csv
 fi
 
-# Set up things we need to update runtime and hibike
+# Set up directories needed by other projects
 mkdir -p $HOME/updates
-cp $FRANKFURTER_DIR/resources/update.sh $HOME/bin/
-
 mkdir -p $HOME/bin
-cp PieCentral/DevOps/frankfurter/resources/mac.py $HOME/bin/
-
 mkdir -p $HOME/studentcode
 
-chmod +x $HOME/bin/*
+# Copy executables
+cp $FRANKFURTER_DIR/resources/update.sh $HOME/bin
+cp $FRANKFURTER_DIR/resources/mac.py $HOME/bin
+cp $FRANKFURTER_DIR/resources/runtime.sh $HOME/bin
 
-# copy .conf files into /etc/init so that hibike/dawn/runtime start on boot
-sudo cp $FRANKFURTER_DIR/resources/memcached.conf /etc
-sudo cp $FRANKFURTER_DIR/resources/runtime.sh $HOME/bin/
+# Configure executable permissions
+sudo chown ubuntu $HOME/bin/*
+sudo chgrp ubuntu $HOME/bin/*
+sudo chmod 755 $HOME/bin/*
+
+# Create and enable the `runtime` and `update` systemd services
 sudo cp $FRANKFURTER_DIR/resources/runtime.service /lib/systemd/system
+sudo cp $FRANKFURTER_DIR/resources/update.service /lib/systemd/system
 sudo chmod 644 /lib/systemd/system/runtime.service
-sudo systemctl daemon-reload && sudo systemctl enable runtime.service
+sudo chmod 644 /lib/systemd/system/update.service
+sudo systemctl daemon-reload
+sudo systemctl enable update.service
+sudo systemctl enable runtime.service
+sudo systemctl start runtime.service
 
-# copy config files for grizzlies, memcached, and network interfaces
+# Copy config files for memcached and network interfaces
+sudo cp $FRANKFURTER_DIR/resources/memcached.conf /etc
 sudo cp $FRANKFURTER_DIR/resources/interfaces /etc/network/interfaces
 
-echo 'export PYTHONPATH="${PYTHONPATH}:/home/ubuntu/PieCentral/hibike"' >> $HOME/.bashrc  # .profile?
+echo 'export PYTHONPATH="${PYTHONPATH}:/home/ubuntu/PieCentral/hibike"' >> $HOME/.bashrc
+
+# Aliases for runtime development
```

Comment:
If this is here, it shouldn't be hardcoded in the bashrc file
Choose one or the othe